mir-ci/mir_ci/wayland/protocols/virtual_keyboard_unstable_v1/zwp_virtual_keyboard_manager_v1.py

```diff
@@ -0,0 +1,95 @@
+# This file has been autogenerated by the pywayland scanner
```

Comment:
Is this file supposed to be committed?

Reply:
Oops, these files were being used only by the test example. They can be removed.